spek-ide-plugin-intellij-base/src/main/kotlin/org/spekframework/intellij/inspections/NoArgConstructorInspection.kt

```diff
@@ -1,7 +1,3 @@
-/**
```

Comment:
just some style cleanup. No @authors allowed, git should be the source of truth for this information